apps/ledger-live-mobile/src/families/elrond/accountActions.tsx

```diff
@@ -37,8 +36,7 @@ const getMainActions = ({
   /*
    * Get a list of all the providers, randomize, and also the screen, conditionally, based on existing amount of delegations.
    */
-  const preloaded = getCurrentElrondPreloadData();
-  const validators = randomizeProviders(preloaded.validators);
+  const validators = useElrondRandomizedValidators();
```

Comment:
Here is a commit that show how to go around this error: https://github.com/LedgerHQ/ledger-live/commit/ecc416c30e8cc76cd4da90652fd4c7cb6fb776c5#diff-0a10fcd15ed4ded6fa6691960e4fbb6e5c6c1784d5670392d0068161c8322f27

Reply:
Note that it also means that coin-elrond had to be added as a dependency in the ledger-mobile package.json. https://github.com/LedgerHQ/ledger-live/commit/ecc416c30e8cc76cd4da90652fd4c7cb6fb776c5#diff-337fdf074b180711a3604b65e063ce0985e61b9d4a78e95c8634d46e3036ad04

Reply:
I redid a bit of QA, launched mobile app and did a send, the changes I've commited work fine.